.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -4,24 +4,55 @@ about: Request our thoughts on your suggestion for a new feature for Homebrew.
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+<details>
+<summary><strong>Guidelines</strong></summary>
+Thank you for making a feature suggestion! :heart:
 
-<!-- Please fill these sections with the relevant information: -->
+**Please fill out all the sections below with the relevant information.**
 
-# A detailed description of the proposed feature
+We will review your suggestion and let you know our decision as soon as we can.
+Note, that:
 
-<!-- replace me -->
+- proposed feature must be relevant to at least 90% of Homebrew users.
+- we will close this issue unless maintainers actively plan on working
+  on it or making it `help wanted`.
 
-# The motivation for the feature
+Homebrew is a non-profit project run entirely by unpaid volunteers.
+We, therefore, reserve the right to close your issue without comment if you
+delete, do not read, or do not fill out the issue each section below and provide
+_**all** information requested_. If you repeatedly fail to fill out the issue template,
+you will be blocked from ever submitting issues to Homebrew.
 
-<!-- replace me -->
+Thank you for your understanding,<br>
+\- Homebrew maintainers
+</details>
 
-# How the feature would be relevant to at least 90% of Homebrew users
 
-<!-- (if it's not: do not open a feature request) -->
+## Description
+_A detailed description of the proposed feature._
 
-# What alternatives to the feature have been considered
+replace me
```

Comment:
This should be a hidden comment, like before.

Reply:
I hid these back but... why should these be hidden? From the UX perspective, it seems that having these shown by default should make it easier for people to edit.

Reply:
> From the UX perspective, it seems that not having HTML comment tags should make the template a bit easier for people to edit.

I disagree. I'd really like if this PR can be scoped on fixing problems _we've actually seen contributors have_ rather than speculating on what is or isn't easy.